player/lua/console.lua

```diff
@@ -20,6 +20,7 @@ local opts = {
     font = "",
     font_size = 16,
     border_size = 1,
+    scale_with_window = false,
```

Comment:
vidscale is inaccurate because it doesn't scale with the video but with window, and vidscale itself is inconsistent with "osd-scale-by-window" anyway. They should all have been called scale with window. I don't think we need to keep using the bad naming from osc and stats in other scripts.